src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+                            .stripLeading());
+                    list.add(curr);
+                    System.out.println(topBound + "Got it! I've added this task:\n" + "  " + curr
+                            + "\nNow you have " + list.noOfTasks() + " tasks in the list." + bottomBound);
+                } catch (DukeException e) {
+                    // command came without a description
+                    System.out.println(topBound + e.getMessage() + bottomBound);
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    System.out.println(topBound + "deadline must include '/by time'. Try again!" + bottomBound);
+                }
+            } else if (arr[0].startsWith("event")) {
+                try {
+                    String taskName = arr[0].replaceFirst("event", "").stripLeading().stripTrailing();
+                    if (taskName.equals("")) {
+                        throw new DukeException("event");
+                    }
+                    Events curr = new Events(taskName, arr[1].replaceFirst("at", "").stripLeading());
+                    list.add(curr);
+                    System.out.println(topBound + "Got it! I've added this task:\n" + "  " + curr
+                            + "\nNow you have " + list.noOfTasks() + " tasks in the list." + bottomBound);
+                } catch (DukeException e) {
+                    // command came without a description
+                    System.out.println(topBound + e.getMessage() + bottomBound);
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    System.out.println(topBound + "event must include '/at duration'. Try again!" + bottomBound);
+                }
+            } else if (arr[0].startsWith("delete")) {
+                try {
+                    String str = arr[0].replaceFirst("delete", "").stripLeading();
+                    int x = Integer.parseInt(str) - 1;
+                    Task curr = list.get(x);
+                    list.delete(x);
+                    System.out.println(topBound + "Got it! I've removed this task:\n" + " " + curr + "\n" + "You have "
+                            + list.noOfTasks() + " tasks left in the list." + bottomBound);
+                } catch (IndexOutOfBoundsException e) {
+                    // task number x does not exist
+                    System.out.println(topBound + "Task does not exist and cannot be deleted!" + bottomBound);
+                } catch (NumberFormatException e) {
+                    // parameter for delete is not an Integer
+                    System.out.println(topBound + "delete must be followed by an Integer, ie. delete 1. Try again!"
+                        + bottomBound);
+                }
+            } else {
+                // unknown command
+                System.out.println(topBound + "OOPS!!! I'm sorry, but I don't know what that means :-(" + bottomBound);
+            }
```

Comment:
Since there are many if else statements, I think using switch would make the code look cleaner :)